src/app/styles/base/_settings.scss

```diff
@@ -4,9 +4,15 @@ $error: #a52a2a;
 $light-grey: #eaeaea;
 $mid-grey: #646464;
 $dark-grey: #5c5c5c;
+$background-dark: #1f1f1f;
+$background-promotion: #161616;
+$orange: #f58550;
+$black: #000;
 
 //Font Size
 $font-size-small: 0.9rem;
+$font-size-med: 1.2;
+$font-size-big: 2.4rem;
```

Comment:
Maybe you should use the same font size units here

Reply:
this is a typo, thanks. I will make this change